docs/content/en/docs/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -19,7 +19,7 @@ they appear in this example.
 
 ## Stage 1
```

Comment:
Thanks for making scripts to simplify Edge Syncer bootstrapping. Looks good to me for the parts relating with Edge Syncer (`mailbox-prep.sh`, `ensure-syncer-plugin.sh`). 

I went through the steps of the latest example1.md and found a minor typo at L703. The destination is not `florin` but `guilder`.

> Also check out the SinglePlacementSlice objects in
`root:my-org:wmw-s`.  It should go similarly, but the `destinations`
should include only the entry for florin.

`the destinations should include only the entry for florin.` 
=>
`the destinations should include only the entry for guilder.`

Here is the actual result of my end about SPS in wmw-s.
```
$ kubectl ws root:my-org:wmw-s            
Current workspace is "root:my-org:wmw-s".
```
```
$ kubectl get SinglePlacementSlice -o yaml
apiVersion: v1
items:
- apiVersion: edge.kcp.io/v1alpha1
  destinations:
  - cluster: 1v06m7rbr96l6fp9
    locationName: guilder
    syncTargetName: guilder
    syncTargetUID: a119906a-7144-4265-af99-bc65fd7b40a3
  kind: SinglePlacementSlice
  metadata:
    annotations:
      kcp.io/cluster: jvpco3mwpeixuarv
    creationTimestamp: "2023-04-26T03:12:54Z"
    generation: 1
    name: edge-placement-s
    ownerReferences:
    - apiVersion: edge.kcp.io/v1alpha1
      kind: EdgePlacement
      name: edge-placement-s
      uid: 765b5eec-0cca-42c3-b426-cad39697518b
    resourceVersion: "1384"
    uid: 0cb13942-7e0e-40b6-8a74-c56bd5bfc3b1
kind: List
metadata:
  resourceVersion: ""
```

Reply:
Oops, thanks.  Fixed in the latest push.

Reply:
Thanks. /lgtm

Reply:
Thanks.  FYI, "/lgtm" only works if it is at the start of a line.